lib/pages/chat/chat_pinned_events/pinned_messages_screen.dart

```diff
@@ -71,109 +71,177 @@ class PinnedMessagesScreen extends StatelessWidget {
+                                                          .primary,
+                                                  BlendMode.srcIn,
+                                                ),
+                                              ),
+                                            if (action.iconData != null)
+                                              Icon(
+                                                action.iconData,
+                                                color: action.color,
+                                              ),
+                                            if (action.imagePath != null ||
+                                                action.iconData != null)
+                                              PinnedMessagesStyle
+                                                  .paddingIconAndUnpin,
+                                            Text(
+                                              action.text,
+                                              style: Theme.of(context)
+                                                  .textTheme
+                                                  .labelLarge
+                                                  ?.copyWith(
+                                                    color: action.color,
+                                                  ),
+                                            ),
+                                          ],
+                                        ),
+                                      ),
                                     ),
-                                  ),
-                                ),
-                              )
-                              .toList(),
+                                  )
+                                  .toList(),
+                            );
+                          },
                         );
                       },
-                      childCount: controller.eventsNotifier.value.length,
+                      childCount: events.length,
                     ),
                   );
                 },
               ),
             ),
           ),
-          if (responsiveUtils.isMobile(context)) ...[
-            Divider(
-              height: 0.5,
-              color: Theme.of(context).colorScheme.secondaryContainer,
-            ),
-            SizedBox(
-              height: PinnedMessagesStyle.unpinButtonHeight,
-              width: MediaQuery.of(context).size.width,
-              child: TextButton(
-                child: Row(
-                  mainAxisAlignment: MainAxisAlignment.center,
-                  crossAxisAlignment: CrossAxisAlignment.center,
-                  children: [
-                    SvgPicture.asset(
-                      ImagePaths.icUnpin,
-                      colorFilter: ColorFilter.mode(
-                        Theme.of(context).primaryColor,
-                        BlendMode.srcIn,
+          ValueListenableBuilder<List<Event>>(
```

Comment:
The UI is for web only, so why don't you remove the `responsiveUtils.isMobile(context)`

Reply:
it's the same window as mobile, I couldn't find the design for mobile so I just adapted it a little bit by removing names on buttons 

![image](https://github.com/linagora/twake-on-matrix/assets/48354990/1e7153f2-40cf-48bc-a278-58a0810acca9)

Reply:
the mobile can do later, for now, you can ignore it in mobile